### PR TITLE
better support for scala 2.13

### PR DESCRIPTION
**Pull request by kailuowang**

### CODE REVIEW

File: src/main/scala/sbtcatalysts/CatalystsPlugin.scala [hunk 1]
```diff
@@ -522,8 +523,8 @@ trait CatalystsBase {
   lazy val partialUnificationSettings = Seq(
     scalacOptions ++= {
       CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, 10)) => Nil  //Xling:unused warn against unused implicit evidence
-        case _ => Seq("-Ypartial-unification")
+        case Some((2, n)) if Set(11L, 12L).contains(n) => Seq("-Ypartial-unification")
```
Comment (SethTisue) [inline src/main/scala/sbtcatalysts/CatalystsPlugin.scala]: `Some((2, 11 | 12))`

Comment (kailuowang) [reply src/main/scala/sbtcatalysts/CatalystsPlugin.scala]: Ha, learn something new about scala every day

**Comment by kailuowang** [comment]
@SethTisue is that all? thanks for the review, btw.

**Comment by kailuowang** [comment]
I am going to have to merge for now, if you have anything @SethTisue I'll address in a separate PR.

**Comment by kailuowang** [comment]
I am going to have to merge for now, if you have anything @SethTisue I'll address in a separate PR.

**Comment by SethTisue** [comment]
that's all I got

**Pull request merged**